### ISSUE TITLE
Selenium IDE File upload - Error code: RESULT_CODE_KILLED_BAD_MESSAGE

### ISSUE THREAD

**Issue by mar1n3r0** [SeleniumHQ/selenium-ide#1097]
## 🐛 Bug Report

On File upload the test completes with success but the file is not uploaded. The process is interrupted by a popup with the window open, a modal with Open File native window open. At the background the message 

> "Aw snap"

> Error code: RESULT_CODE_KILLED_BAD_MESSAGE

.

## To Reproduce

Enable Allow access to file URLs and do a  file upload via IDE.

## Expected behavior

The file should upload.

## Project file reproducing this issue (highly encouraged)
```
{
  "id": "acff94ff-c39a-4aaa-be6d-69ebda3675d2",
  "version": "2.0",
  "name": "file upload",
  "url": "https://test.local",
  "tests": [{
    "id": "f4ec5b75-050c-43c3-b199-cdb6a12ec8cc",
    "name": "file upload 2",
    "commands": [{
      "id": "3a189b09-5eb1-4629-9fdc-5462eab40da0",
      "comment": "",
      "command": "open",
      "target": "/",
      "targets": [],
      "value": ""
    }, {
      "id": "6aa549ae-d2b8-465c-92e4-c2e982a89e5e",
      "comment": "",
      "command": "setWindowSize",
      "target": "1919x1046",
      "targets": [],
      "value": ""
    }, {
      "id": "0b7eb67b-4062-4715-b54b-e5f33e4011bb",
      "comment": "",
      "command": "type",
      "target": "id=file",
      "targets": [
        ["id=file", "id"],
        ["name=file", "name"],
        ["css=#file", "css:finder"],
        ["xpath=//input[@id='file']", "xpath:attributes"],
        ["xpath=//form[@id='uploadForm']/div[5]/input", "xpath:idRelative"],
        ["xpath=//div[5]/input", "xpath:position"]
      ],
      "value": "C:\\fakepath\\60484366_2789736551041785_7482132015190376448_n.jpg"
    }, {
      "id": "642f4482-914c-4ae4-8fc3-57682ddb38a1",
      "comment": "",
      "command": "click",
      "target": "css=.form__button",
      "targets": [
        ["css=.form__button", "css:finder"],
        ["xpath=//button[@type='submit']", "xpath:attributes"],
        ["xpath=//form[@id='uploadForm']/button", "xpath:idRelative"],
        ["xpath=//button", "xpath:position"]
      ],
      "value": ""
    }]
  }
```

## Environment

OS: Linux
Selenium IDE Version:  3.17.0
Browser:
Browser Version: Version 83.0.4103.116 (Developer Build) (64-bit)

**Comment by capoerafreak** [comment]
Just to be sure, in your real test you didn‘t wrote: `C:\\fakepath\\...` as value for the type command?

If you record a file upload, the filename is saved but not the real path.

**Comment by mar1n3r0** [comment]
Yes, correct. This is taken from the record so the browser already hid the path.

**Comment by capoerafreak** [comment]
Sorry for my late response. I have no issue with uploading.

Please try my side file (rename it from ".txt" to ".side"), create a file and correct the path in the value field
[Upload.txt](https://github.com/SeleniumHQ/selenium-ide/files/4973357/Upload.txt)

![image](https://user-images.githubusercontent.com/36188654/88416871-44925d80-cde1-11ea-8afd-7c504ace636f.png)

**Comment by mar1n3r0** [comment]
@capoerafreak Thanks for the feedback. I have tried your file and even though the test completes successfully(as with my example) the debugging Chromium window crashes with the same message. It looks like it has something to do with the webdriver since it doesn't crash Selenium IDE but rather the browser debugger session.

![Screenshot_20200725_100743](https://user-images.githubusercontent.com/11375673/88451413-37b44f00-ce5f-11ea-8cb9-88ffafabbee0.png)

**Comment by capoerafreak** [comment]
@mar1n3r0 sorry, that I can not help you ...

**Comment by doanva84** [comment]
@mar1n3r0 : 
I've tried following simple example of @capoerafreak and it works well.

## Steps
1. Create a simple local file-uploader (i.e https://gist.github.com/taterbase/2688850, should add `id="Upload"` to button attribute for eases in click event)
2. Use Firefox/Chrome Selenium IDE to make simple test. For example:
```
Command         Target                          Value
open          /upload.php
type          name=uploaded_file          C:\Temp\auto-test.txt
click            id=Upload
```

3. Ctrl + R to run
* And you could see 'The file auto-test.txt has been uploaded'

Enjoy experiencing!

**Comment by mar1n3r0** [comment]
Thanks for the suggestion @doanva84. Unfortunately this is not suitable for my case where I would like to record already working browser action so that I don't need to do hundreds of uploads by hand every time. In order for this to work only the native Selenium IDE project files work.
